ideascube/utils.py

```diff
@@ -1,8 +1,27 @@
+from io import TextIOWrapper
 import sys
 
 from django.conf import locale
 
 
+class AutomagicTextIOWrapper(TextIOWrapper):
```

Comment:
>> It seems to me that what you're implying is that, for you, the obvious thing to do would have been to splitlines and use an array, which makes using a TextIOWrapper here a sort of special case in your view. Am I right?

> Nope. The obvious thing to do would have been to just give the file to DictReader, but we cannot because… and this is where the comment comes in ;)

Oh, so you want a comment explaining that we need to decode the `bytes` to `str`?

From your first comment, I thought you wanted a comment explaining why we use an iterator-like solution instead of `to_unicode(source.read()).splitlines()`.

I guess this whole conversation was based on a misunderstanding, then.